airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -25,45 +25,42 @@ module.exports = function (plop) {
   const pythonSourceInputRoot = '../source-python';
   const singerSourceInputRoot = '../source-singer';
   const genericSourceInputRoot = '../source-generic';
+  const httpApiInputRoot = '../source-python-cdk';
 
   const outputDir = '../../connectors';
   const pythonSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
   const singerSourceOutputRoot = `${outputDir}/source-{{dashCase name}}-singer`;
   const genericSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
+  const httpApiOutputRoot = `${outputDir}/source-{{dashCase name}}`;
 
   plop.setActionType('emitSuccess', function(answers, config, plopApi){
       console.log(getSuccessMessage(answers.name, plopApi.renderString(config.outputPath, answers), config.message));
   });
 
-  plop.setGenerator('source-python', {
-    description: 'Generate an Airbyte Source written in Python',
-    prompts: [{type: 'input', name: 'name', message: 'Source name, without the "source-" prefix e.g: "google-analytics"'}],
+  plop.setGenerator('Python HTTP API Source', {
```

Comment:
@sherifnada what's the reason for not having 'CDK' in the name? I feel pretty strongly that we should stick to the same terminology throughout.  Is it because we don't want to release this yet?

Reply:
Totally agree. But CDK is basically all of base python. We will rename it to CDK down the road. So calling things CDK seems like overkill? WDYT?

Reply:
I see. I still prefer keeping CDK as 1) using CDK is a nice reminder to people they are using a spanking brand new low-code framework 2) it can be confusing if we use CDK in our other materials and not have it appear in the templates, especially since we have other http apis implemented via the non-CDK stuff 3) having it as a separate option besides all our other options highlights our progress 

I'm a 5/10 on this, so if you feel stronger than me, I gladly defer.

Reply:
I'm going to keep CDK for now - open to changing at a later date :)